webpack.config.js

```diff
@@ -124,27 +124,13 @@ const reactNativeConfig = {
  *
  *   1. This package is designed for Node only and hence requires polyfills of Node libraries (e.g. `stream`, `path`) — webpack takes care of this for us.
  *   2. The package is not compatible with RequireJS and hence we don’t have any easy way to directly load it in our tests — the webpack bundle exposes it as a global named MochaJUnitReporter.
- *
- * - Node: The library uses optional chaining syntax, which is not supported by Node 12.
```

Comment:
Do we need a Node bundle at all any more, or can `test/support/mocha_reporter.js` now just directly `require('mocha_unit_reporter')`? In removing the Node webpack config I think we can also remove the `webpack-node-externals` dependency introduced in dd7c83ea910c580a4ee20f72cf19babd59351b76.

Reply:
Will check